locales/templates/app.pot

```diff
@@ -464,6 +464,14 @@ msgid ""
 "contents from the most popular topics to the hottest deals. %2$s"
 msgstr ""
 
+msgctxt "first_run_page7_title"
+msgid "Prevent from being tracked"
```

Comment:
Please add a localization comment here so we know what "Prevent" refers to (the way you write it will change depending on what it refers to)

Reply:
It means prevent websites from tracking you

Reply:
thank you, please add that as a localization comment so our translators will know what this is about

Reply:
Title message shown in a privacy onboarding page. It means to prevent websites from tracking you.